### DISCUSSION TITLE
Very Low FPS on Carla

### DISCUSSION THREAD

**Discussion by 00Dan00** [carla-simulator/carla#7502] [Q&A]
I'm trying to run Carla on my Dell Precision 3541 where I installed Ubuntu 22.04. I got the newest Carla Version from Github but the Server (and also the  client) runs with very low fps and delayed input without even spawning vehicles or anything. Is there anything I did wrong since I feel like it shouldn't need that many resources.

I also often found that you should disable "Use Less CPU When in Background" in UE4 but since I got it from github I didn't download UE4 so is this not applicable to my case (or should I try to build it from source for better performance?)

Ty for all answers

**Comment by gsmario** [comment]
Hello! Would you be so kind to tell us your computer specs? 
Thanks in advance!
